packages/docs/content/docs/inline-comments.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Inline Comments
+description: Add, view, and manage line-level comments directly on the diff code
+icon: "MessageCircleCode"
+---
+
+# Inline Comments
+
+Inline Comments let you leave feedback directly on the code generated by Pochi inside VS Code. Instead of re-explaining issues in chat, you can comment on specific lines in the diff, batch those comments, and send them back to Pochi for the right contextual fixes.
+
+This feature is not your regular PR review system. It’s actual conversations with the LLM, attached to the code they refer to. 
+
+<video
+        controls
+        style={{
+        width: "100%",
+        borderRadius: "8px",
+        boxShadow: "0 4px 12px rgba(0, 0, 0, 0.15)",
+        }}
+    >
+        <source src="https://assets.docs.getpochi.com/inline-comments.mp4" type="video/mp4" />
+        Your browser does not support the video tag.
+    </video>
+
+
+## How to Use Inline Comments
+
+### 1. Run a Prompt
+Generate code or request changes like normal.
+
+```bash
+/pochi please use Mermaid Chart instead of Vega Lite
+```
+
+### 2. Open the Diff
+Once Pochi applies changes, open the diff view in VS Code:
+![Pochi Models](../assets/images/open-diff-inline-comments.png)
+
+In the diff, hover over a code line and click `+` to add a comment
+  ![Pochi Models](../assets/images/add-comment-inline-comments.png)
+
+
+### 3. Leave Feedback
+Type your comment directly on the line where the issue exists.
+
+![Pochi Models](../assets/images/leave-feedback-inline-comment.png)
+
+
+### 4. Add Multiple Comments
+Leave comments across the file as needed. Each comment is automatically linked to its exact line + file path.
+
+![Pochi Models](../assets/images/multiple-comments-inline-comment.png)
+
+
+### 5. Send Comments as a Batch
+Once ready:
+* Click `Submit Reviews` to Pochi
+* Or run the command: 
```

Comment:
mention that this works just like submit reviews if submitting empty messages, users could add additional messages and submit in chatbox if necessary.